cms/djangoapps/contentstore/views/course.py

```diff
@@ -1137,7 +1137,7 @@ def settings_handler(request, course_key_string):
                             entrance_exam_minimum_score_pct = float(ee_min_score_pct)
                         if entrance_exam_minimum_score_pct.is_integer():
                             entrance_exam_minimum_score_pct = entrance_exam_minimum_score_pct / 100
-                        entrance_exam_minimum_score_pct = unicode(entrance_exam_minimum_score_pct)
+                        entrance_exam_minimum_score_pct = entrance_exam_minimum_score_pct
```

Comment:
This line of code is no longer needed, correct?

Reply:
Duh, correct; removing.